bindings/rust/bench/src/s2n_tls.rs

```diff
@@ -203,6 +260,14 @@ impl TlsConnection for S2NConnection {
         self.connection.actual_protocol_version().unwrap() == Version::TLS13
     }
 
+    fn resumed_connection(&self) -> bool {
+        !self
+            .connection
+            .handshake_type()
+            .unwrap()
+            .contains("FULL_HANDSHAKE")
```

Comment:
Why does this indicate that the session was resumed?

Reply:
Any connection that went through the full handshake will contains the "FULL_HANDSHAKE" designation, and if not it won't be present. This is essentially copied from our `IS_RESUMPTION_HANDSHAKE` check in `s2n_handshake_type.h`.
https://github.com/aws/s2n-tls/blob/5d5400aea80e033d73b0343dd03d915a83aaa549/tls/s2n_handshake_type.h#L29-L30

Reply:
Ah okay, that makes sense. Thanks!